### PR TITLE
I added simple login

### PR DESCRIPTION
**Pull request by person1234-ux**
I added simple login.

### CODE REVIEW

**Comment by person1234-ux** [comment]
all relevant information can be found in https://github.com/2factorauth/twofactorauth/issues/6658 as I couldn't find documentation

File: img/s/simplelogin.io.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  </g>
+</svg>
```
Comment (Carlgo11) [inline img/s/simplelogin.io.svg]: Here's a version with just the logo.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 129.8 76.3"><defs><linearGradient id="a" x1="50.6" x2="150.1" y1="298.8" y2="298.8" gradientTransform="translate(-66 -190.9)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ed2e7c"/><stop offset="1" stop-color="#a8288f"/></linearGradient></defs><path fill="url(#a)" d="M126.2 0H32.7l-4.2 4.2c-9 7-17.4 12.1-27 14.5a49 49 0 0 0 27.8 56.5l2.6 1h94.3c2 0 3.6-1.6 3.6-3.6v-69c0-2-1.6-3.6-3.6-3.6zm-6.4 6-39 34a2 2 0 0 1-2.9 0L65.1 28.8c-.2-3.8-.4-6.6-1.2-10A60.9 60.9 0 0 1 39.4 6h80.4zM67.2 38 64 40.9l.8-5zM32 72A45.3 45.3 0 0 1 4.8 22a63.6 63.6 0 0 0 28-16 63.7 63.7 0 0 0 27.7 16 46.7 46.7 0 0 1-2.1 24A45.2 45.2 0 0 1 32 72zm11.4-1.5a49 49 0 0 0 16.3-18.3L71.5 42l6.4 5.7c.8.7 2 .7 2.8 0l6.5-5.8 32.6 28.7zM91.5 38l32.6-29 .1 57.7ZM55 25.1C46.5 23 39 18 32.7 12a50.1 50.1 0 0 1-22.5 13.4c-.4 2.3-.6 4.6-.6 6.9 2.7 15.8 9.7 28.6 22.6 34A37.9 37.9 0 0 0 55.6 32a43 43 0 0 0-.6-6.6zM31 52 17.2 38.2l4.2-4.2 9.6 9.5 15-15 4.2 4.3C43.7 39.1 37.5 45.8 31 52z"/></svg>
```

File: entries/s/simplelogin.io.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Simple Login": {
```
Comment (Carlgo11) [inline entries/s/simplelogin.io.json]: The correct spelling seems to be as one word.
```suggestion
  "SimpleLogin": {
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by person1234-ux** [comment]
I am going to clone this because my fork seems to be corrupted and I don't want to corrupt this one.  I will reopen when I know it is safe to do so

**Pull request closed without merging**